src/index/DeltaTriples.cpp

```diff
@@ -86,26 +110,64 @@ DeltaTriplesCount DeltaTriples::getCounts() const {
   return {numInserted(), numDeleted()};
 }
 
+// _____________________________________________________________________________
+DeltaTriples::Triples DeltaTriples::makeInternalTriples(
+    const Triples& triples) {
+  Triples internalTriples;
+  for (const auto& triple : triples) {
+    const auto& ids = triple.ids();
+    Id objectId = ids.at(2);
+    auto optionalLiteralOrIri = ExportQueryExecutionTrees::idToLiteralOrIri(
+        index_, objectId, localVocab_);
```

Comment:
(Some thinking aloud) We start with TripleComponents from the parser, lookup their IDs and now resolve the IDs back to the literal Values. That is a bit strange. But I also don't see an obvious place to do this where everything is present in TripleComponents. Variables are only bound to the value from the result once everything has been translated to IDs.